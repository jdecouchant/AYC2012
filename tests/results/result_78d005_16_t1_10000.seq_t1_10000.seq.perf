
Elapsed real time                  0:00.00
CPU-seconds spent in kernel        0.00
CPU-seconds spent in user          0.01
Percentage of the CPU got          237%
Maximum resident set size          3668 Kbytes.
Average resident set size          0 Kbytes.
data+stack+text memory use         0 Kbytes.
Average size unshared data area    0 Kbytes.
Average unshared stack space       0 Kbytes.
Average size shared text space     0 Kbytes.
Number of major page faults        0
Number of recoverable page faults  566
Number of context-switch           13
Number of waits                    78
Number of file system inputs       0
Number of file system outputs      8
