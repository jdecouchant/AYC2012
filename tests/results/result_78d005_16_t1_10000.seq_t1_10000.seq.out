Sequence NÂ°0
1 10000 1 10000
